src/app/weather-forecast/weather-forecast.component.ts

```diff
@@ -0,0 +1,22 @@
+import {Component, OnInit} from '@angular/core';
+import { WeatherForecastApiService } from './weatherForecastApiService/weather-forecast-api.service';
+
+@Component({
+    selector: 'app-weather-forecast',
+    templateUrl: './weather-forecast.component.html',
+    styleUrls: ['./weather-forecast.component.scss'],
+})
+export class WeatherForecastComponent implements OnInit {
+    response: object;
+    constructor(private wfas: WeatherForecastApiService) {}
+
+    ngOnInit() {
+        this.retriveWF();
+    }
```

Comment:
do you really need to do this in the `OnInit` handler? Can  it be moved to constructor?